widgets/SuccessFactors-batchsource.json

```diff
@@ -0,0 +1,124 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "SAP SuccessFactors",
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName",
+          "widget-attributes": {
+            "placeholder": "Used to uniquely identify this source for lineage, annotating metadata etc."
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "SAP SuccessFactors Base URL",
+          "name": "baseURL",
+          "widget-attributes": {
+            "placeholder": "SAP SuccessFactors base url e.g. https://<hostname>/odata/v2"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Entity Name",
+          "name": "entityName",
+          "widget-attributes": {
+            "placeholder": "SAP SuccessFactors Entity name. Eg. People"
+          },
+          "plugin-function": {
+            "method": "POST",
+            "widget": "outputSchema",
+            "plugin-method": "getSchema"
+          }
+        }
+      ]
+    },
+    {
+      "label": "Credentials",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "SAP SuccessFactors Logon Username",
+          "name": "username",
+          "widget-attributes": {
+            "placeholder": ""
+          }
+        },
+        {
+          "widget-type": "password",
+          "label": "SAP SuccessFactors Logon Password",
+          "name": "password",
+          "widget-attributes": {
+            "placeholder": ""
+          }
+        }
+      ]
+    },
+    {
+      "label": "Advanced",
+      "properties": [
+        {
+          "widget-type": "textarea",
+          "label": "Filter Options",
+          "name": "filterOption",
+          "widget-attributes": {
+            "placeholder": "Eg. Price gt 200"
+          }
+        },
+        {
+          "widget-type": "textarea",
+          "label": "Select Fields",
+          "name": "selectOption",
+          "widget-attributes": {
+            "placeholder": "Eg. Category,Price,Name,Supplier/Address"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Expand Fields",
+          "name": "expandOption",
+          "widget-attributes": {
+            "placeholder": "Eg. Products,Products/Suppliers"
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Number of Rows to Skip",
```

Comment:
Please update config based on our offline discussion

Reply:
Removed the properties: `Number of Rows to Skip` and `Number of Rows to Fetch` 444c10c0add0efdb6799cd258c03b10d1f479f5e